.env

```
@@ -0,0 +1,3 @@
+REACT_APP_API_BASE_URL=https://survey-api.nimblehq.co
+REACT_APP_API_CLIENT_ID=6GbE8dhoz519l2N_F99StqoOs6Tcmm1rXgda4q__rIw
+REACT_APP_API_CLIENT_SECRET=_ayfIm7BeUAhx2W1OUqi20fwO3uNxfo1QstyKlFCgHw
```

Please add a newline at the end of file 🙏 
```suggestion
REACT_APP_API_CLIENT_SECRET=_ayfIm7BeUAhx2W1OUqi20fwO3uNxfo1QstyKlFCgHw

```